packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -923,30 +921,30 @@ class SearchDropdown<T> extends StatelessWidget {
 
 class _AutoCompleteSearchField extends StatefulWidget {
   const _AutoCompleteSearchField({
-    @required this.searchField,
-    @required this.controller,
-    @required this.searchFieldFocusNode,
-    @required this.rawKeyboardFocusNode,
-    @required this.autoCompleteLayerLink,
-    @required this.onSelection,
-    @required this.onHighlightDropdown,
-    @required this.clearSearchField,
+    required this.searchField,
+    required this.controller,
+    required this.searchFieldFocusNode,
+    required this.rawKeyboardFocusNode,
+    required this.autoCompleteLayerLink,
+    required this.onSelection,
+    required this.onHighlightDropdown,
```

Comment:
if this is nullable, let's make it optional

Reply:
made them non nullable

Reply:
It's probably better to let this be optional since `onHighlightDropdown` is nullable. Another argument for making this optional - the only place that `_AutoCompleteSearchField` is constructed is in `buildAutoCompleteSearchField`, where `onHighlightDropdown` is an optional parameter.